src/Query/QueryGenerator.php

```diff
@@ -52,7 +53,7 @@
      *
      * @param Connection             $connection
      * @param QueryPlatformInterface $queryPlatform
-     * @param QueryField[]           $fields
+     * @param array     $fields
```

Comment:
All CS will be fixed after merging as there are also other CS problems that need to be resolved.